package.json

```diff
@@ -21,7 +21,7 @@
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-config-prettier": "^9.0.0",
     "eslint-plugin-import": "^2.29.0",
-    "eslint-plugin-json": "^3.1.0",
+    "eslint-plugin-jsonc": "^2.14.1",
```

Comment:
Replacing the plugin is a good thing, but would also require a change in the ESLint configuration.

I see two options here:

1. adjust the ESLint configuration accordingly. Or
2. undo the change and we address this in a new PR. As I also like and use this module, I am quite willing to do this :slightly_smiling_face: